mono/mini/mini-ops.h

```diff
@@ -1382,7 +1382,7 @@ MINI_OP(OP_PPC_FLOOR,              "ppc_floor", FREG, FREG, NONE)
 MINI_OP(OP_PPC_TRUNC,              "ppc_trunc", FREG, FREG, NONE)
 #endif
 
-#if defined(TARGET_ARM) || defined(TARGET_ARM64)
+#if defined(TARGET_ARM) || defined(TARGET_ARM64) || defined(TARGET_AMD64)
```

Comment:
I don't think `TARGET_AMD64` should enable this code path.

Reply:
so the reason for this is because i have to build the genmdesc project for windows x64 to build arm64 files which means the TARGET_ARM64 is not set

so the build fails as those are not defined

Reply:
I think this maybe the distinction between HOST_ARM64 (platform the executable will run on) vs TARGET_ARM64 (target for machine code generated by the executable).

Mono wasn't always precise with this in past, especially on Windows, but this may be the distinction needed. I'll take a closer look tomorrow.